extensions/dacpac/src/wizard/dataTierApplicationWizard.ts

```diff
@@ -332,6 +334,7 @@ export class DataTierApplicationWizard {
 	}
 
 	private async extract(): Promise<mssql.DacFxResult> {
+		let additionalProps: TelemetryEventProperties = {};
```

Comment:
Might as well just define this all in one :

```
const additionalProps: TelemetryEventProperties = {
   version: this.model.version
}
```	

could also just inline it into the function call itself.